run-test.sh

```diff
@@ -5,8 +5,9 @@
 : ${ENCRYPTED_PATH:=/data/encrypted}
 
 source /app/functions.sh
+source /app/s3sync.sh
 
-set -e;
```

Comment:
If `set -e` is removed, will the `verify_test_file` below cause the script to exit if it fails?